.stylelintrc.json

```diff
@@ -1,6 +1,3 @@
 {
-  "extends": "stylelint-config-recommended",
-  "rules": {
-    "block-closing-brace-newline-after": "always"
-  }
+  "extends": "stylelint-config-recommended"
```

Comment:
Deprecated rule: https://stylelint.io/migration-guide/to-15/#deprecated-stylistic-rules